### DISCUSSION TITLE
How to rerun failed tests?

### DISCUSSION THREAD

**Discussion by rafaelmereb** [junit-team/junit-framework#3142] [Q&A] [answered]
Apparently, @RepeatedTest does not execute conditionally. 
Is there a way to rerun a failed test until the result is a success? or even for a maximum number of attemps?

**Comment by sbrannen** [comment, marked as answer]
No, that functionality does not currently exist in JUnit Jupiter.

## Related Issues

- #1558
- #2119
- #2925
